testing/functional_tests/test_segment_volume.py

```diff
@@ -79,7 +80,8 @@ def test_segment_volume_2d(download_functional_test_files):
     shutil.rmtree(PATH_MODEL)
 
 
-def test_segment_volume_3d(download_functional_test_files):
+@pytest.mark.parametrize("length_3d", [[192, 192, 16], LENGTH_3D, ])
```

Comment:
```suggestion
@pytest.mark.parametrize("center_crop", [[192, 192, 16], LENGTH_3D])
```
I think we could only test for [192, 192, 16] here, if it works for 4 sub-volumes, then it will work for 1.

Reply:
Great point. I will just simplify it then to make it more readable. I kept the old one as I wasn't sure if there was anything special/unique about the old value.